hal/targets/hal/TARGET_STM/TARGET_STM32F7/rtc_api.c

```diff
@@ -39,10 +40,33 @@ static int rtc_inited = 0;
 
 static RTC_HandleTypeDef RtcHandle;
 
+#if DEVICE_RTC_LSI
```

Comment:
Reserve DEVICE_ for device_has list. other macros can use STM_XXX or just RTC_LSI